app/code/Magento/Newsletter/Controller/Subscriber/Confirm.php

```diff
@@ -15,11 +17,11 @@
 class Confirm extends \Magento\Newsletter\Controller\Subscriber implements HttpGetActionInterface
 {
     /**
-     * Subscription confirm action.
+     * Subscription confirm action
      *
      * @return \Magento\Framework\Controller\Result\Redirect
      */
-    public function execute()
+    public function execute(): \Magento\Framework\Controller\Result\Redirect
```

Comment:
Unfortunately, we can't change public methods signature according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
@sidolov fixed.